### PR TITLE
[bsp] 解决N9H30 HMI开发板触摸屏误触发

### PR DESCRIPTION
**Pull request by mysterywolf**
https://club.rt-thread.org/ask/article/ccbb3e920a9ad23c.html

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [ ] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [ ] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [ ] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [ ] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality
- [ ] 本拉取/合并使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](../documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](../documentation/contribution_guide/coding_style_en.txt)

### CODE REVIEW

File: bsp/nuvoton/libraries/nu_packages/ADC_TOUCH/adc_touch.c [hunk 1]
```diff
@@ -640,8 +641,8 @@ static void adc_touch_entry(void *parameter)
     }
 
     if (rt_memcmp((void *)&g_sCalMat, (void *)&g_sCalZero, sizeof(S_CALIBRATION_MATRIX)) != 0)
-        g_u32Calibrated = 1;
 
+    g_u32Calibrated = 1;
```
[the remark this answers is not in the document]
Comment (mysterywolf) [reply bsp/nuvoton/libraries/nu_packages/ADC_TOUCH/adc_touch.c:645]: 唔 感谢指正

**Comment by andychen329** [comment]
@wosayttn pls revew

**Comment by wosayttn** [comment]
感謝 @andychen329 @thewon86 @mysterywolf 
我 review 下，PR 方案不完整 ( ADC_TOUCH_FILTER_VAL -> Where is the definition?)
我手邊並無法覆現這個現象。我先做些測試/實驗，優化看看。後續我再提交。

**Comment by mysterywolf** [comment]
OK

**Pull request closed without merging**